docs/community_resources/code.md

```diff
@@ -0,0 +1,19 @@
+# Contribute code
+
+Every intention to change existing code or introduce new one should start with a proper explanation. It should be placed in a public ticket. All the rules which you should follow can be found in [contribute documentation section](documentation.md).
+
+Once you are done with describing your idea, focus on the main part - sharing the actual solution. In eZ we are using regular git workflow, so if you are familiar with the concept, the whole procedure should be pretty straightforward.
+
+As you probably know, eZ Platform is divided into meta-repositories. The core of our system is [ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel) containing an advanced Content Model and aiming to provide additional features for the MVC layer (Symfony). On the other hand, e.g. `ezplatform-admin-ui` is strictly dedicated for the Admin Panel purposes. If you want to learn more about our code structure, take a look at [organization page on GitHub](https://github.com/ezsystems) or the list of [core bundles](https://doc.ezplatform.com/en/latest/guide/bundles/#core-bundles). 
+
+After finishing your work, you are free to fork repository which you want to contribute to. Now you need to determine which version of the package should be targeted by your changes. If you plan to fix something in your current project, check `composer.json` for version of the package and pick proper branch. Let's follow the example: you managed to add `try { } catch () {}` statement fixing some annoying error in `ezplatform-admin-ui` and you are using eZ Platform 2.5. You should aim for branch `1.5` then as version `1.5.0` is used. 
+
+Now you can follow the procedure is exactly the same as in [Contributing through git](documentation/#contributing-through-git).
+
+!!! caution "Public repositories"
+
+    You can contribute only to the public repositories what means that all repositories marked as `private` are not open to contributions outside our organization.
```

Comment:
maybe add (very briefe) mention here about generating patch and uploading it as part for ticket on support.ez.no as a way to contribute on private/ee packages.

Reply:
Added in: https://github.com/ezsystems/developer-documentation/pull/617/commits/6ff2dab6f94185b6ca4a8afc4adce2216d674f0c.